webpack.config.js

```diff
@@ -254,25 +253,14 @@ if ( process.env.DASHBOARD ) {
 	webpackConfig.plugins.unshift( new DashboardPlugin() );
 }
 
-if ( process.env.WEBPACK_OUTPUT_JSON ) {
+if ( ! isDevelopment ) {
 	webpackConfig.devtool = 'cheap-module-source-map';
 	webpackConfig.plugins.push(
-		new webpack.optimize.UglifyJsPlugin( {
-			minimize: true,
-			compress: {
-				warnings: false,
-				conditionals: true,
-				unused: true,
-				comparisons: true,
-				sequences: true,
-				dead_code: true,
-				evaluate: true,
-				if_return: true,
-				join_vars: true,
-				negate_iife: false,
-				screw_ie8: true,
-			},
-			sourceMap: true,
+		new UglifyJsPlugin( {
+			cache: true,
+			parallel: true,
+			uglifyOptions: { ecma: 5 },
```

Comment:
do we need any of the config bits from the old config? `screw_ie8` and friends?

Reply:
so i looked up the [defaults](https://github.com/webpack-contrib/uglifyjs-webpack-plugin/tree/master#uglifyoptions) for both `mangle` and `compress`

I don't remember my discoveries line by line, but i remember the defaults catching everything we were specifying manually. For example, ie8 support is disabled by default.